all_msgs/ros2interop/ros-types-all_connector.xml

```diff
@@ -0,0 +1,1482 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
An example that uses connector, can simply include the file with types above. It'd be something like...

```
<dds>
    <types>
        <include file="..."/>
    </types>
</dds>
```

But I don't think that example belongs in this folder. We could add it to a different repository or document it in a README.md file within this folder.

Reply:
Using <include ...> with connector; I assume it needs a preprocessor to handle this tag..?
Otherwise it reports an error.
Agree that it should be put elsewhere -- not in this folder.